File: content/en/docs/Tutorials/Emitter and Receiver/LOP.py
if receiver.getQueueLength() > 0: # If receiver queue is not empty
    receivedData = receiver.getData()
    tup = struct.unpack('c', receivedData) # Parse data into character
    if tup[0].decode("utf-8") == 'L': # 'L' means lack of progress occurred
        print("Detected Lack of Progress!") 
    receiver.nextPacket() # Discard the current data packet

File: content/en/docs/Tutorials/Emitter and Receiver/LOP.c
if(receiver->getQueueLength() > 0) { // If receiver queue is not empty
    char *message = (char *)receiver->getData(); // Grab data as a string
    if (message[0] == 'L') { // 'L' means a lack of progress occurred
        cout << "Detected Lack of Progress!" << endl;
        receiver->nextPacket(); // Discard the current data packet
    }
}